lib/taurus/qt/qtgui/panel/taurusmodelchooser.py

```diff
@@ -29,24 +29,120 @@
 from __future__ import print_function
 from __future__ import absolute_import
 
+from builtins import bytes, str
+
 import sys
-from taurus.external.qt import Qt
+import pkg_resources
+
+import taurus
+from taurus.external.qt import Qt, QtCore
 import taurus.core
 from taurus.qt.qtgui.container import TaurusWidget
 from taurus.qt.qtgui.tree import TaurusDbTreeWidget
 from taurus.core.util.containers import CaselessList
 from .taurusmodellist import TaurusModelList
 
+__all__ = ["TaurusModelSelectorTree", "TaurusModelChooser",
+           "TaurusModelSelector", "TaurusModelSelectorItem",
+           "TangoModelSelectorItem"]
```

Comment:
Since the location of the `TangoModelSelectorItem` is not yet definitive, maybe we could skip including it in `__all__` for now, and just register it for the entrypoint using the whole module path in `<taurus>/setup.py`

Reply:
Done